test/example.c

```diff
@@ -558,9 +571,11 @@ int main(argc, argv)
     } else if (strcmp(zlibVersion(), ZLIB_VERSION) != 0) {
         fprintf(stderr, "warning: different zlib version\n");
     }
-
-    printf("zlib version %s = 0x%04x, compile flags = 0x%llx\n",
-            ZLIB_VERSION, ZLIB_VERNUM, zlibCompileFlags());
+    /* ensure flags is 64 bit regardless whether uLong is 32 or 64 bit
+     */
+	uint64_t flags64 = zlibCompileFlags();
+    printf("zlib version %s = 0x%" PRIx64 ", compile flags = 0x%llx\n",
```

Comment:
Inconsistent. Need yo replace all %ll with PRI. I also assumed that %04x was OK?